utils/sort.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import "strconv"
+
+type NumericSort []string
+
+func (a NumericSort) Len() int      { return len(a) }
+func (a NumericSort) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+func (a NumericSort) Less(i, j int) bool {
```

Comment:
Do we want this sort algorithm to be public? Or should it go under an `internal/` package?

Reply:
IMO the iso8583 is not a sotring algorithm, so it kinda make sense to put it into the internal.

Reply:
I named the package to `internal/sorting`, because I wanted to avoid the collision with the built-in sort naming. Let me know if you have better idea for the naming.